pkg/services/folder/folderimpl/sqlstore.go

```diff
@@ -325,3 +326,34 @@ func (ss *sqlStore) GetHeight(ctx context.Context, foldrUID string, orgID int64,
 	}
 	return height, nil
 }
+
+func (ss *sqlStore) GetFolders(ctx context.Context, orgID int64, uids []string) ([]*folder.Folder, error) {
+	var folders []*folder.Folder
+	if err := ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+		b := strings.Builder{}
+		args := make([]any, 0, len(uids)+1)
+
+		b.WriteString("SELECT * FROM folder WHERE org_id=? ")
+		args = append(args, orgID)
+		for i, uid := range uids {
+			if i == 0 {
+				b.WriteString("  AND (")
+			}
+
+			if i > 0 {
+				b.WriteString(" OR ")
```

Comment:
I think I can use `IN (...)` here :)

Reply:
based on [this] discussion yes: for some backends IN() is just a synonym for multiple OR clauses but MySQL seems to do better: by [using binary search](https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#operator_in)